packages/salesforcedx-vscode-core/test/vscode-integration/commands/baseDeployRetrieve.test.ts

```diff
@@ -569,62 +569,6 @@ describe('Base Deploy Retrieve Commands', () => {
         expect(appendLineStub.calledOnce).to.equal(true);
         expect(appendLineStub.firstCall.args[0]).to.equal(expectedOutput);
       });
-
-      xit('should report any diagnostics if deploy failed', async () => {
```

Comment:
Are we trying to test the same behavior in 'should unlock queue on failure'?

Reply:
I just realized I was missing a case. Thanks!